detoxify/core.py

```diff
@@ -1,12 +1,13 @@
 import torch
 import transformers
 
+DOWNLOAD_URL = "https://github.com/unitaryai/detoxify/releases/download/"
```

Comment:
this is mainly to make the lines shorted and so please flake8 and 120 line length